scripts/installer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:8
+FROM centos/centos:stream8
```

Comment:
Can you please use quay.io/centos/centos:stream8 so it always pulls from quay and not whatever is the default on the machine it's executed on?

Reply:
done!